storage/storage.go

```diff
@@ -435,14 +435,14 @@ func (s *Storage) recoverNullifiers() error {
 	// Recover nullifiers from aggregated batches
 	aggregatedBallotsReader := prefixeddb.NewPrefixedReader(s.db, aggregBatchPrefix)
 	if err := aggregatedBallotsReader.Iterate(nil, func(k, v []byte) bool {
-		// Decode the aggregated ballot batch to extract nullifiers
+		// Decode the aggregated ballot batch to extract voteIDs
 		abb := &AggregatorBallotBatch{}
 		if err := DecodeArtifact(v, abb); err != nil {
 			outerErr = fmt.Errorf("failed to decode aggregated ballot batch: %w", err)
 			return false // Stop iterating on error
 		}
 		for _, ballot := range abb.Ballots {
-			s.lockNullifier(ballot.VoteID.BigInt().MathBigInt())
+			s.lockVoteID(ballot.VoteID.BigInt().MathBigInt())
```

Comment:
I think we should also lock by nullifier. I mean, while we are processing a nullifier, we might not accept another vote for the same nullifier.

Reply:
This PR only contains some updates in names that were wrong. If we want to also lock by nullifier, it is out of the scope of this PR and we should do it in a new one.